diff --git a/node_modules/@fujocoded/zod-transform-socials/dist/index.cjs b/node_modules/@fujocoded/zod-transform-socials/dist/index.cjs
index bcc56c2..7dca638 100644
--- a/node_modules/@fujocoded/zod-transform-socials/dist/index.cjs
+++ b/node_modules/@fujocoded/zod-transform-socials/dist/index.cjs
@@ -204,11 +204,11 @@ var extractSocialData = createExtractSocialData(socialLinks);
 
 // index.ts
 var SocialsSchema = import_zod.default.union([
-  import_zod.default.string().url(),
+  import_zod.default.url(),
   import_zod.default.object({
     icon: import_zod.default.string().optional(),
     platform: import_zod.default.string().optional(),
-    url: import_zod.default.string().url(),
+    url: import_zod.default.url(),
     username: import_zod.default.string().optional()
   })
 ]);
diff --git a/node_modules/@fujocoded/zod-transform-socials/dist/index.js b/node_modules/@fujocoded/zod-transform-socials/dist/index.js
index d816b62..8bdf65d 100644
--- a/node_modules/@fujocoded/zod-transform-socials/dist/index.js
+++ b/node_modules/@fujocoded/zod-transform-socials/dist/index.js
@@ -1,8 +1,8 @@
+// social-links.ts
+import { SocialLinks as SocialLinksLib } from "social-links";
 // index.ts
 import z from "zod";
 
-// social-links.ts
-import { SocialLinks as SocialLinksLib } from "social-links";
 var DOMAIN_PATTERNS = {
   mastodon: (domain) => ({
     match: `https?://${escapeForRegex(domain)}/@([a-z0-9-_]+)/?`,
@@ -167,11 +167,11 @@ var extractSocialData = createExtractSocialData(socialLinks);
 
 // index.ts
 var SocialsSchema = z.union([
-  z.string().url(),
+  z.url(),
   z.object({
     icon: z.string().optional(),
     platform: z.string().optional(),
-    url: z.string().url(),
+    url: z.url(),
     username: z.string().optional()
   })
 ]);
